### PR TITLE
Frameworks and Preprocessors lesson: Move Knowledge Check section above Additional Resources section

### PR DESCRIPTION
**Pull request by DevVivan**
## Because

I have moved the section titled 'Knowledge Check' above the 'Additional Resources' section to match the layout of previous lessons in The Odin Project. Not sure if this positioning was intentional, but I believe this matches the lesson layout from the official style guide much better.

## This PR

- Moved 'Knowledge check' section above 'Additional resources' section to match layout style guide.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: intermediate_html_css/intermediate_css_concepts/frameworks_and_preprocessors.md [hunk 1]
```diff
@@ -64,3 +57,10 @@ This section contains questions for you to check your understanding of this less
 * [What are some disadvantages of using a CSS framework?](https://www.smashingmagazine.com/2018/11/css-frameworks-css-grid/)
 * [What are some advantages of using a CSS preprocessor?](https://medium.com/@LiliOuakninFelsen/what-is-a-css-pre-processor-why-should-you-use-one-ac3d4cd4b2c2)
 * [What are some disadvantages of using a CSS preprocessor?](https://adamsilver.io/blog/the-disadvantages-of-css-preprocessors/)
+
+### Additional Resources
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/frameworks_and_preprocessors.md:61]:
```suggestion
### Additional Resources

```

Comment (ManonLef) [reply intermediate_html_css/intermediate_css_concepts/frameworks_and_preprocessors.md:61]: Small thing to edit: all headings should have [a newline before and after it](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#newlines)

Thanks for making this PR

**Review by ManonLef — CHANGES_REQUESTED** [review]
This one is ready to merge if the heading issue is fixed

**Comment by DevVivan** [comment]
Oh, yes. So sorry about that, my bad, I'll work on it now.

**Comment by DevVivan** [comment]
Yep, completed, so the new commit titled 'Move Knowledge Check section above Additional Resources section and fix spacing' fixes the spacing issue

**Comment by CouchofTomato** [comment]
@ManonLef Can you take another look at this one please.

**Review by ManonLef — APPROVED** [review]

**Comment by ManonLef** [comment]
Thank you @DevVivan !

**Pull request merged**